src/public/loading-error.html

```diff
@@ -7,18 +7,21 @@
 	<meta http-equiv="X-UA-Compatible" content="IE=edge">
 	<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no" />
 	<link rel="stylesheet" type="text/css" href="../stylesheets/main.css">
+    <script>
+        const i18n = require('../i18n/index');
```

Comment:
I'm not a super fan of this solution, but since we have just a few html files, it will do.